components/constellation/constellation.rs

```diff
@@ -3007,7 +3002,7 @@ where
         let browsing_context = match self.browsing_contexts.get(&browsing_context_id) {
             Some(bc) => bc,
             None => {
-                warn!("BC has closed before it has started");
+                warn!("browsing context has closed before it has started");
```

Comment:
For consistency with the other messages this would start with a capital letter no? Maybe this should be prefixed with the id like the other messages instead?

Reply:
done in 9772e8a36bd55a3619c20199e5081cfee8491cab